src/traces/sankey/render.js

```diff
@@ -155,7 +155,7 @@ function linkModel(uniqueKeys, d, l) {
     var tc = tinycolor(l.color);
     var basicKey = l.source.label + '|' + l.target.label;
     var foundKey = uniqueKeys[basicKey];
-    uniqueKeys[basicKey] = (foundKey === void(0) ? foundKey : 0) + 1;
+    uniqueKeys[basicKey] = (foundKey === void(0) ? 0 : foundKey) + 1;
```

Comment:
might be clearer if we just use the fact that `foundKey` will always be truthy if it exists?
```
uniqueKeys[basicKey] = (foundKey || 0) + 1;
var key = basicKay + (foundKey ? '' : ('__' + foundKey));
```

Which brings up: it might be very uncommon, but what if the user actually had a `label` like `'name'` and another like `'name__1'`? I could see this popping out of some upstream disambiguation... Wouldn't it be more robust to always add the suffix like:
```
var keyCounter = uniqueKeys[basicKey] || 0;
uniqueKeys[basicKey] = keyCounter + 1;
var key = basicKey + '_' + keyCounter;
```